tests/manager_tests/test_vulnerabilities_handler.py

```diff
@@ -184,19 +184,19 @@ def test_vulnerabilities_affecting_filter(self):
 
         response1 = self.vfetch('vulnerabilities/cves?affecting=true,false').check_response()
         response2 = self.vfetch('vulnerabilities/cves/ids?affecting=true,false').check_response()
-        assert len(response1.body.data) == len(response2.body.data) == 17
+        assert len(response1.body.data) == len(response2.body.data) == 18
```

Comment:
should `affecting=true,false`, or `affecting=false` return edge systems?

Reply:
I think it shouldn't

Reply:
@blayson This API should not return edge systems under any parameter.

Reply:
Yeah, thanks. Just curious, it's already done :)